ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ethernet_interfaces.schema.yml

```diff
@@ -979,6 +979,149 @@ keys:
         port_profile:
           type: str
           description: Key only used for documentation or validation purposes
+        uc_tx_queues:
+          type: list
+          primary_key: id
+          convert_types:
+            - dict
+          items:
+            type: dict
+            keys:
+              id:
+                type: int
+                convert_types:
+                  - str
+                required: true
+                description: TX-Queue ID
+              random_detect:
+                type: dict
+                keys:
+                  ecn:
+                    description: Explicit Congestion Notification
+                    type: dict
+                    keys:
+                      count:
+                        type: bool
+                        description: |
+                          Enable counter for random-detect ECNs
+                      threshold:
+                        type: dict
+                        keys:
+                          units:
+                            required: true
+                            type: str
+                            valid_values: ["segments", "bytes", "kbytes", "mbytes"]
+                            description: |
+                              Indicate the units to be used for the threshold values.
+                              This should be one of segments, byte, kbytes, mbytes.
+                          min:
+                            type: int
+                            required: true
+                            min: 1
+                            max: 256000000
+                            convert_types:
+                              - str
+                            description: |
+                              Set the random-detect ecn minimum-threshold
+                          max:
+                            type: int
+                            required: true
+                            convert_types:
+                              - str
+                            min: 1
+                            max: 256000000
+                            description: |
+                              Set the random-detect ecn maximum-threshold
+                          max_probability:
+                            type: int
+                            min: 1
+                            max: 100
+                            convert_types:
+                              - str
+                            description: |
+                              Set the random-detect ecn max-mark-probability.
+                              The value should be between 1 and 100.
```

Comment:
```suggestion
```
This information is already contained in min,max and doesn't need to be repeated. This should be corrected through the proposed schema. Also for valid values etc.

Reply:
Agreed - removing such references in descriptions.